File: Contributions/simpleCalculator.py
#github : https://github.com/arincantikam26

('=' * 25)
print('simple calculator')
print(' 1. addition\t\t[+]')
print(' 2. subtraction\t\t[-]')
print(' 3. multiplication\t[*]')
print(' 4. division\t\t[/]')
print('=' * 25)

    data = input('Choice operation(1/2/3/4): ')
    data_1 = eval(input('Input first number : '))
    data_2 = eval(input('Input second number : '))
    
    if data == '1':
        print('addition\t\t[+]\n')
        result = data_1 + data_2
        print(f'Result : {data_1} + {data_2} = {result}')
    elif data == '2':
        print('subtraction\t\t[-]\n')
        result = data_1 - data_2
        print(f'Result : {data_1} - {data_2} = {result}')
    elif data == '3':
        print('multiplication\t[*]\n')
        result = data_1 * data_2
        print(f'Result : {data_1} * {data_2} = {result}')
    elif data == '4':
        print('division\t\t[/]')
        result = data_1 / data_2
        print(f'Result : {data_1} / {data_2} = {result}')
    else :
        print('Invalid option')

    


    
    

File: Dart/simple_calculator.dart
import 'dart:io';

void main() {
  print("Simple Calculator");
  print("Enter the first number: ");
  double num1 = double.parse(stdin.readLineSync()!);

  print("Enter the second number: ");
  double num2 = double.parse(stdin.readLineSync()!);

  print("Select an operation (+, -, *, /): ");
  String operation = stdin.readLineSync()!;

  double result;

  switch (operation) {
    case '+':
      result = num1 + num2;
      break;
    case '-':
      result = num1 - num2;
      break;
    case '*':
      result = num1 * num2;
      break;
    case '/':
      if (num2 != 0) {
        result = num1 / num2;
      } else {
        print("Error: Division by zero.");
        return;
      }
      break;
    default:
      print("Invalid operation.");
      return;
  }

  print("Result: $num1 $operation $num2 = $result");
}
